Don't call file with seccomp in portage sandbox. This will fail:

  make[2]: *** [Makefile:834: magic.mgc] Bad system call

--- file-5.39/magic/Makefile.am
+++ file-5.39/magic/Makefile.am
@@ -352,5 +352,5 @@
 		exit 1; \
 	    fi; \
 	  fi)
-	$(FILE_COMPILE) -C -m magic
+	$(FILE_COMPILE) -S -C -m magic
 	@rm -fr magic
